gradient/commands/common.py

```diff
@@ -214,3 +207,22 @@ def _log_logs_continuously(self, id, line, limit):
     def _get_logs_generator(self, id, line, limit):
         logs_gen = self.client.yield_logs(id, line, limit)
         return logs_gen
+
+    def _make_table(self, logs, id):
+        table_title = "%s %s logs" % (self.ENTITY, id)
+        table_data = [("LINE", "MESSAGE")]
+        table = terminaltables.AsciiTable(table_data, title=table_title)
+
+        for log in logs:
+            table_data.append(self._format_row(log))
+
+        return table.table
+
+    def _get_log_row_string(self, id, log):
+        log_msg = "{}\t{}".format(*self._format_row(log))
+        return log_msg
+
+    @staticmethod
+    def _format_row(log_row):
+        return (style(fg="red", text=str(log_row.line)),
+                log_row.message)
```

Comment:
Why is this moved to the end of the class? You could have implemented those methods without moving them around.

Reply:
Doing changes this way makes it harder to read git file history